bitcoin.py

```diff
@@ -37,3 +37,26 @@ def btc(bot, trigger):
 
     except Exception:
         return bot.say("Failed to check the bitstamp price")
+
+@commands('bch')
+def bch(bot, trigger):
+    """Tells information about bitcoin cash"""
+
+    command = trigger.group(2)
+
+    if not command:
+        command = price
+
+    try:
+        bytes = web.get('https://api.bitfinex.com/v2/candles/trade:1m:tBCHUSD/last')
```

Comment:
Probably better to use something longer than 1m. Most interested in the last price, anyway, but others may want to know the swing.